extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -410,11 +416,13 @@ class LifecycleTask extends BackbeatTask {
      * For all filtered rules, get rules that apply the earliest
      * @param {array} rules - list of filtered rules that apply to a specific
      *   object, version, or upload
+     * @param {boolean} isLCUser - is current account a lifecycle service
+     *   account
      * @return {object} all applicable rules with earliest dates of action
      *  i.e. { Expiration: { Date: <DateObject>, Days: 10 },
      *         NoncurrentVersionExpiration: { NoncurrentDays: 5 } }
      */
-    _getApplicableRules(rules) {
+    _getApplicableRules(rules, isLCUser) {
```

Comment:
I would rather not shorten the variable name to win a few chars (so `isLifecycleUser`), but this may conflict with the helper function name (maybe?), so we can keep in doubt.

Reply:
maybe since its a boolean, i can call it something like `defaultEnable`? or something along those lines?